.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+# Description
+Briefly explain what all changes you have made. Don't forget to mention any dependencies that are added in this change.
+
+Fixes # (issue number)
+
+## Type of change
+Please delete options that are not relevant.
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+## Checklist:
+- [ ] My changes generate no new warnings
+- [ ] This PR follows the contribution guide
+- [ ] I have performed a self-review of my own code
+- [ ] My code follows the style guidelines of this project
+- [ ] I have made corresponding changes to the documentation
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] Any dependent changes have been merged and published in downstream modules
+
+## Screenshot
+Add screenshots of UI changes or any new implementations you have made. Incase of no visual change remove this section.
+
+**Mentions**
+- @technojam/app-development 
```

Comment:
Any particular reason of adding the mentions?

Reply:
Just to ping the team members, nothing specific can be removed. 
What do you suggest?

Reply:
I have made the necessary changes just let me know if we should keep Mentions or not. Then I'll commit the new changes.

Reply:
I think we can use the assign and review sections for that. It'll send everyone an email.